src/content/learn/referencing-values-with-refs.md

```diff
@@ -93,12 +93,12 @@ export default function Stopwatch() {
   const [now, setNow] = useState(null);
 
   function handleStart() {
-    // Start counting.
+    // Saymaya başla
     setStartTime(Date.now());
     setNow(Date.now());
 
     setInterval(() => {
-      // Update the current time every 10ms.
+      // "now" state değişkenini her 10ms'de günceller. 
```

Comment:
I think we can keep the comments similar since they aren't ment for explaining the flow that detailed:
```suggestion
      // Şuanki zamanı her 10ms'de bir güncelle. 
```